robo.yml

```diff
@@ -9,49 +9,49 @@ backend:
-    url: "http://publications.europa.eu/resource/cellar/b8924e2d-ce0b-11eb-ac72-01aa75ed71a1.0001.04/DOC_1"
+    url: "http://publications.europa.eu/resource/cellar/947c85d5-581c-11ec-91ac-01aa75ed71a1.0001.01/DOC_1"
     format: "rdf"
 
   - name: "corporate-body"
     graph: "http://publications.europa.eu/resource/authority/corporate-body"
-    url: "http://publications.europa.eu/resource/cellar/1e09ca83-594b-11ec-91ac-01aa75ed71a1.0001.03/DOC_1"
+    url: "http://publications.europa.eu/resource/cellar/dca0f07d-787e-11ec-9136-01aa75ed71a1.0001.03/DOC_1"
     format: "rdf"
 
   - name: "country"
     graph: "http://publications.europa.eu/resource/authority/country"
-    url: "http://publications.europa.eu/resource/cellar/acb33658-ce0b-11eb-ac72-01aa75ed71a1.0001.04/DOC_1"
+    url: "http://publications.europa.eu/resource/cellar/820161ea-581c-11ec-91ac-01aa75ed71a1.0001.04/DOC_1"
     format: "rdf"
 
   - name: "digital-europa-thesaurus"
     graph: "http://data.europa.eu/uxp"
-    url: "http://publications.europa.eu/resource/cellar/b451491c-ce0b-11eb-ac72-01aa75ed71a1.0001.03/DOC_1"
+    url: "http://publications.europa.eu/resource/cellar/76166330-581c-11ec-91ac-01aa75ed71a1.0001.03/DOC_1"
     file: "DET-skos-core.rdf"
     format: "zip"
 
   - name: "eurovoc-thesaurus"
     graph: "http://publications.europa.eu/resource/dataset/eurovoc"
-    url: "http://publications.europa.eu/resource/cellar/b868cf85-c47b-11eb-a925-01aa75ed71a1.0001.04/DOC_1"
+    url: "http://publications.europa.eu/resource/cellar/bcd714b1-5f05-11ec-9c6c-01aa75ed71a1.0001.04/DOC_1"
     file: "eurovoc_in_skos_core_concepts.rdf"
     format: "zip"
 
   - name: "eu-programme"
     graph: "http://publications.europa.eu/resource/authority/eu-programme"
-    url: "http://publications.europa.eu/resource/cellar/acaf65c7-ce0b-11eb-ac72-01aa75ed71a1.0001.02/DOC_1"
+    url: "http://publications.europa.eu/resource/cellar/d11bd217-2070-11ec-bd8e-01aa75ed71a1.0001.02/DOC_1"
     format: "rdf"
 
   - name: "language"
     graph: "http://publications.europa.eu/resource/authority/language"
-    url: "http://publications.europa.eu/resource/cellar/a6c51d84-ce0b-11eb-ac72-01aa75ed71a1.0001.04/DOC_1"
+    url: "http://publications.europa.eu/resource/cellar/87f03e0d-581c-11ec-91ac-01aa75ed71a1.0001.04/DOC_1"
     format: "rdf"
 
   - name: "organization-type"
     graph: "http://publications.europa.eu/resource/authority/organization-type"
-    url: "http://publications.europa.eu/resource/cellar/af196871-1c71-11e8-ac73-01aa75ed71a1.0001.01/DOC_1"
+    url: "http://publications.europa.eu/resource/cellar/d0a4a644-2070-11ec-bd8e-01aa75ed71a1.0001.04/DOC_1"
     format: "rdf"
 
   - name: "place"
     graph: "http://publications.europa.eu/resource/dataset/place"
-    url: "http://publications.europa.eu/resource/cellar/a6c125e3-ce0b-11eb-ac72-01aa75ed71a1.0001.03/DOC_1"
+    url: "http://publications.europa.eu/resource/cellar/9d7afcb9-8684-11eb-ac4c-01aa75ed71a1.0001.01/DOC_1"
```

Comment:
According to https://op.europa.eu/en/web/eu-vocabularies/dataset/-/resource?uri=http://publications.europa.eu/resource/dataset/place&version=20211208-0, the latest skos-ap-act file is http://publications.europa.eu/resource/cellar/87ec948c-581c-11ec-91ac-01aa75ed71a1.0001.03/DOC_1